doc/api/assert.md

```diff
@@ -147,6 +147,136 @@ try {
 }
 ```
 
+## Class: assert.CallTracker
+
+### `new assert.CallTracker()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Creates a new [`CallTracker`][] object which can be used to track if functions
+were called a specific number of times. The `tracker.verify()` must be called
+for the verification to take place. The usual pattern would be to call it in the
+`process` `exit` handler.
```

Comment:
```suggestion
for the verification to take place. The usual pattern would be to call it in a
handler for the [`process` `'exit'` event][].
```

And then you'll also need to add this at the bottom of the file:

```markdown
[`process` `'exit'` event]: process.html#process_event_exit
```

Reply:
Or maybe this?

```suggestion
[`process.on('exit')`][] handler.
```

That would correspond with usage added in the errors.md file in this PR.